doc/changelog.md

```diff
@@ -78,49 +90,63 @@
-  >>> "Inside QNode: 0.5"
+  "Inside QNode: 0.5"
+  Array(0.77015115, dtype=float64)
   ```
 
-* Autograph now supports in-place array assignments with static slices. [(#843)](https://github.com/PennyLaneAI/catalyst/pull/843)
+* Autograph now supports in-place array assignments with static slices.
+  [(#843)](https://github.com/PennyLaneAI/catalyst/pull/843)
 
   For example,
 
   ```python
   @qjit(autograph=True)
   def f(x, y):
-    y[1:10:2] = x
-    return y
+      y[1:10:2] = x
+      return y
   ```
 
   ```pycon
   >>> f(jnp.ones(5), jnp.zeros(10))
-  >>> Array([0., 1., 0., 1., 0., 1., 0., 1., 0., 1.], dtype=float64)
+  Array([0., 1., 0., 1., 0., 1., 0., 1., 0., 1.], dtype=float64)
   ```
 
-* Autograph works when `qjit` is applied to a function decorated with `vmap`, `cond`, `for_loop` or `while_loop`.
+* Autograph now works when `qjit` is applied to a function decorated with `vmap`, `cond`, `for_loop` or `while_loop`.
   [(#835)](https://github.com/PennyLaneAI/catalyst/pull/835)
   [(#938)](https://github.com/PennyLaneAI/catalyst/pull/938)
   [(#942)](https://github.com/PennyLaneAI/catalyst/pull/942)
 
+  ```python
+  dev = qml.device("lightning.qubit", wires=2)
+
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      return qml.expval(qml.PauliZ(0))
+  ```
+
+  ```pycon
+  >>> x = jnp.array([0.1, 0.2, 0.3])
+  >>> qjit(vmap(circuit), autograph=True)(x)
+  Array([0.99500417, 0.98006658, 0.95533649], dtype=float64)
+  ```
+
 <h3>Breaking changes</h3>
-* Return values are `jax.Array` typed instead of `numpy.array`.
+
+* Return values of qjit-compiled functions are now of type `jax.Array` instead of `numpy.ndarray`.
+  This should have minimal impact, but code that depends on the output of qjit-compiled function
+  being NumPy arrays will need to be updated.
   [(#895)](https://github.com/PennyLaneAI/catalyst/pull/895)
 
 <h3>Bug fixes</h3>
 
-* Make Autograph copy `QNode` instead of creating new one from scratch to preserve information such as transforms and `mcm_method`. [(#900)](https://github.com/PennyLaneAI/catalyst/pull/900)
+* Fixes a bug where Catalyst would fail to apply quantum transforms and preserve QNode configuration settings when Autograph was enabled.
+  [(#900)](https://github.com/PennyLaneAI/catalyst/pull/900)
   
-* Using float32 in callback functions would not crash in compilation phase anymore,
-  but rather raise the appropriate type exception to the user.
+* Passing arrays with `float32` dtype to callback functions will no longer cause compilation to
+  crash, but rather will raise the appropriate type exception.
   [(#916)](https://github.com/PennyLaneAI/catalyst/pull/916)
 
-* Fix tracing of `SProd` operations
-  [(#935)](https://github.com/PennyLaneAI/catalyst/pull/935)
-
-  After some changes in PennyLane, `Sprod.terms()` returns the terms as leaves
-  instead of a tree. This means that we need to manually trace each term and
-  finally multiply it with the coefficients to create a Hamiltonian.
```

Comment:
I moved this to internal changes, since this bug will never really be user facing. That is, this is an update we made to ensure Catalyst continues to work with v0.38 of PL, but never really was a bug with stable catalyst releases.